bodhi/server/templates/update.html

```diff
@@ -48,6 +48,51 @@
 
     // These are the required taskotron tests
     var requirements = ${update.requirements_json | n};
+    // show the Greenwave decision
+    var greenwave_api_url = '${request.registry.settings["greenwave_api_url"]}';
+    var missing_tests = {};
+    // handle Greenwave decision
+    var handle_unsatisfied_reqs = function(data){
+        $.each(data['unsatisfied_requirements'], function(i, req) {
+            if (req.type == 'test-result-missing') {
+                if (missing_tests[req.item] === undefined)
+                    missing_tests[req.item]= [];
+                missing_tests[req.item].push(req);
+            }
+            // the user may have already specified this in the required taskotron tests
+            if ($.inArray(req.testcase, requirements) == -1) {
+                requirements.push(req.testcase);
+            }
+        });
+    };
```

Comment:
Because I want to show unsatisfied requirements on the web UI which we do not store it in the database.  This is important to let people know why the update is blocked.  Yeah, the first negative point is valid and I'm not sure how to handle this. Should we add a new field into the update table to store unsatisfied requirements? For the second negative point, I think it would not put more load on the Greenwave server since there will be only one request to be sent.